### PR TITLE
More inspectors

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

**Review by raghu67 — APPROVED** [review]
Great job as usual. I don't have any additional suggestions

File: pkg/inspectors/signaturealgorithm/signaturealgorithm_test.go [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+				Username: "someRandomUser",
+				Request:  certificateRequestBytes,
+			},
+		}
+		message, err := inspector.Inspect(client, &request)
+		expectedMessage := ""
+		if !expectAllow {
+			expectedMessage = fmt.Sprintf("SignatureAlgorithm is %s", signatureAlgorithm)
+		}
+		assert.Equal(t, expectedMessage, message, "SignatureAlgorithm", signatureAlgorithm)
```
Comment (raghu67) [inline pkg/inspectors/signaturealgorithm/signaturealgorithm_test.go]: I may be misunderstanding how the assert package works. But looking at the source code it looks like it does a Sprintf into the first argument after message passing all the remaining arguments.  If so, Shouldn't it be "SignatureAlgorithm %s", signatureAlgorithm ?

Comment (johngmyers) [reply pkg/inspectors/signaturealgorithm/signaturealgorithm_test.go]: Good catch. Pushed fixup commits.

**Review by raghu67 — APPROVED** [review]

**Review by raghu67 — APPROVED** [review]

**Comment by johngmyers** [comment]
Squashed first fixup commit and merged

**Pull request closed without merging**